prepare-release.sh

```diff
@@ -54,6 +54,8 @@ if [ "$PROJECT" == "orm" ]; then
 		-Pgradle.publish.key=$PLUGIN_PORTAL_USERNAME -Pgradle.publish.secret=$PLUGIN_PORTAL_PASSWORD \
 		-PhibernatePublishUsername=$OSSRH_USER -PhibernatePublishPassword=$OSSRH_PASSWORD \
 		-DsigningPassword=$RELEASE_GPG_PASSPHRASE -DsigningKeyFile=$RELEASE_GPG_PRIVATE_KEY_PATH
+elif [ "$PROJECT" == "reactive" ]; then
+	# Hibernate Reactive does these checks in the `cirelease` task (called by publish.sh)
```

Comment:
Are you sure nothing gets done before `cirelease` gets executed? In particular there's no tag pushed to the git repo?

The point of having separate `releasePrepare`/`release` tasks was that a failure during preparation would not require reverting anything, in particular deleting tags.